deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/DeviceParser.java

```diff
@@ -113,28 +121,27 @@ public String toJson()
             this.lastActivityTimeString = ParserUtility.getDateStringFromDate(this.lastActivityTime);
         }
 
-        //Codes_SRS_DEVICE_PARSER_34_001: [This method shall return a json representation of this.]
         return gson.toJson(this);
     }
 
     /**
-     * Empty constructor
+     * Empty constructor.
      */
     public DeviceParser()
     {
     }
 
     /**
-     * Constructor for a DeviceParser object that is built from the provided json.
-     * @param json the json to build the object from
-     * @throws IllegalArgumentException if the provided json is null, empty, or not the expected format
+     * Constructor for a DeviceParser object that is built from the provided JSON.
+     *
+     * @param json The JSON to build the object from.
+     * @throws IllegalArgumentException If the provided JSON is {@code null}, empty, or not the expected format.
      */
     public DeviceParser(String json)
     {
         if (json == null || json.isEmpty())
         {
-            //Codes_SRS_DEVICE_PARSER_34_005: [If the provided json is null or empty, an IllegalArgumentException shall be thrown.]
-            throw new IllegalArgumentException("The provided json cannot be null or empty");
+            throw new IllegalArgumentException("The provided json cannot be null or empty.");
```

Comment:
I updated this file to have consistent exception messages, including punctuation.